controllers/Playlist.controller.js

```diff
@@ -0,0 +1,130 @@
+const playlistDb = require("../models/playlist.model");
+const userDb = require("../models/user.model");
+
+module.exports.getAllPlaylists = async (req, res) => {
+  const { userId } = req.params;
+  try {
+    const user = await userDb.findById(userId);
+    console.log("usr Playlist", user.playlists);
+    const { playlists } = await (
+      await userDb.findById(userId)
+    ).execPopulate({ path: "playlists", populate: { path: "videos" } });
```

Comment:
Hey, I am unable to understand why we need to `await` statements. Can you please explain?